### ISSUE TITLE
Translated solution does not show up after translation approval

### ISSUE THREAD

**Issue by MMMAAANNN** [Codewars/codewars.com#99]
Some time ago I translated this kata:
http://www.codewars.com/kata/last-digit-of-a-large-number/revisions
into Python.

Recently, I edited my translation and added Python-specific block there.

Now apparently Jake intended to approve my translation:
http://www.codewars.com/kata/last-digit-of-a-large-number/revisions

But it actually resulted in added Python version which had no solution and no test cases! While my translation had them all:
http://www.codewars.com/kumite/5553c51ed54e7f9b6b000117?sel=5553c51ed54e7f9b6b000117

My translation is currently in "Closed translations" section, so it was supposed to be approved I guess.

I have contributor privileges, but I will not edit it for some time, so that you can see how the bug looks. I'll add solution and test cases as soon as you say it is okay to edit that.

**Comment by jhoffner** [comment]
Not sure I follow. Python has a solution and test cases. Does it not show up for you at all?

**Comment by MMMAAANNN** [comment]
Some more info:

I opened 'Solutions' for Python, and it says 'There are no solutions yet for this kata.'

After that I went to http://www.codewars.com/kata/5511b2f550906349a70004e1/edit/python
... to see if my solution is there or not.

There was a popup about recent changes, should I discard them or keep. I clicked "Keep". After that I so empty blocks for everything in Python.

Now I tried the same link, but clicked "Discard", and those blocks all appeared.

However, this http://www.codewars.com/kata/last-digit-of-a-large-number/solutions still shows "There are no solutions yet for this kata".
